components/capture-build-scan-url-maven-extension/build.gradle.kts

```diff
@@ -20,6 +20,6 @@ java {
     }
 }
 
-tasks.withType<JavaCompile>() {
+tasks.compileJava {
```

Comment:
This is not the same as before. Previously we were configuring all JavaCompile tasks. After this change we're just configuring the task with name `compileJava` but not `compileTestJava`. Whether or not this is what we want I can't say.

Reply:
> After this change we're just configuring the task with name compileJava but not compileTestJava

Thanks Benedikt! Do you know if `tasks.withType` is configuration-avoidance compatible? The docs about this particular method are somewhat ambiguous.

Reply:
I actually don't know. What I know is that `withType() {}` is not lazy, but `withType().configureEach {}` is. I think it's best to ask this question in #bt-questions.